sourcegraph/sourcegraph
# upgrade browser extension to manifest v3

Chrome's and the Chrome Web Store's push for [Manifest V3](https://developer.chrome.com/docs/extensions/develop/migrate) means that our browser extension will start to be unusable in a few months unless we upgrade to Manifest V3.

- Removes "Enable Sourcegraph on this domain" context menu item that didn't seem to work. We were adding this ourselves (using `webext-domain-permission-toggle`), and it didn't seem to work. Our options popup shows a permission request, which is sufficient.
- Otherwise preserves all existing functionality as best I could tell.

## Test plan

1. Confirm that the browser extension continues to inject the Sourcegraph icon on https://github.com/hashicorp/errwrap.
2. Load up https://gitlab.com/sqs/web in my local dev Sourcegraph instance and confirm that the browser extension injects the Sourcegraph icon on https://gitlab.com/sqs/web code files.
